mmaction/models/recognizers/base.py

```diff
@@ -237,20 +237,22 @@ def forward(self,
             raise RuntimeError(f'Invalid mode "{mode}". '
                                'Only supports loss, predict and tensor mode')
 
-    def convert_to_datasample(self, inputs: SampleList,
-                              data_samples: InstanceList) -> SampleList:
-        """Convert predictions to ``ActionDataSample``.
+    def wrap_in_datasample(self, data_samples: SampleList,
```

Comment:
I think this part can be moved into `predict()`. No post-processing or formatting is involved, so I guess there is no need to define a method.

Reply:
that's good